src/CameraScreen/CameraKitCameraScreenBase.js

```diff
@@ -318,10 +321,11 @@ const styles = StyleSheet.create(_.merge(styleObject, {
     alignItems: 'center',
   },
   captureButton: {
-    flex: 1,
     alignSelf: 'center',
     alignItems: 'center',
-    justifyContent: 'center'
+    justifyContent: 'center',
+    width: 66,
+    height: 66
```

Comment:
Why is it better then `flex:1`?

Reply:
TBH I just copied over the code from https://github.com/wix/react-native-camera-kit/pull/83

Reply:
I prefer not  to use magic numbers, I'll upload a fix very soon, thanks!